Models/articles.models.js

```diff
@@ -71,3 +71,24 @@ exports.selectAllArticles = (
     return rows;
   });
 };
+
+exports.updateArticleVoteById = (article_id, updateBody) => {
+  const validUpdate = ["inc_vote"];
+  if (
+    !Object.keys(updateBody).every((key) => {
```

Comment:
As you have designed this endpoint to only update votes, this is a little overkill. However if you were to expand this patch to update any part of the article then I can see a good use case for this.

Reply:
Absolutely. I made this ticket like that way because I think maybe we'll need to expand our patch that's why I did setup for next ones.